### DISCUSSION TITLE
Is the number of edges of dataset "KarateClub" correct?

### DISCUSSION THREAD

**Discussion by PolarisRisingWar** [pyg-team/pytorch_geometric#2683] [Q&A] [answered]
I find from [torch_geometric.datasets.karate — pytorch_geometric 1.7.0 documentation](https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/datasets/karate.html#KarateClub) that this class is to download dataset from NetworkX and handle it.  
The number of edges in NetworkX dataset [karate_club_graph](https://networkx.org/documentation/stable/reference/generated/networkx.generators.social.karate_club_graph.html) has 78 edges (get by code: `G = nx.karate_club_graph(); print(G.number_of_edges());` ) and in [PyG]((https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/datasets/karate.html#KarateClub)) it uses edge_index, so in PyG it should have 156 (2*78) edges (and by code `dataset = KarateClub(); data = dataset[0]; print(data)` I can get `edge_index=[2, 156]`). But in the document and source code of KarateClub, it says that *connected by 154 (undirected and unweighted) edges.*  
So I think it's wrong. If it's really wrong, how could I post that issue? I think it may be less serious than a bug?

**Comment by rusty1s** [comment, marked as answer]
Thanks for spotting. I fixed the typo in documentation. Thank you very much!
